src/SDKs/EdgeGateway/EdgeGateway.Tests/EdgeGateway.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.test.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>Foo</PackageId>
```

Comment:
Please rename this to `EdgeGateway.Tests`

Reply:
Hi @dsgouda 
You have asked to use generate.ps1 (this) to generated client code. But from where it will pick swagger? I think it is picking from https://github.com/Azure/azure-rest-api-specs. We do not have our latest swagger there yet (It is in PR https://github.com/Azure/azure-rest-api-specs/pull/5191). 

I had used below command to generate client code. The client code I get from both are different, is there I way I can specify swagger file to use while generating with the method you mentioned

autorest -CodeGenerator Azure.CSharp -Modeler Swagger -Input "C:\Users\anponnet\Desktop\Petstore\swagger.json" -Namespace Microsoft.Azure.Management.EdgeGateway

Reply:
Fixed

Reply:
Corrected using generate.ps1. Swagger was taken from the fork. https://github.com/anponnet/azure-rest-api-specs/tree/version20190301